book/website/reproducible-research/overview/overview-barriers.md

```diff
@@ -0,0 +1,179 @@
+You can consider computational reproducibility as being necessary but not sufficient for high quality research.
+A critical approach is needed, rather than naively using existing software or implementing statistical methods without understanding what they do.
+See, for example, [a discussion](https://ryxcommar.com/2019/08/30/scikit-learns-defaults-are-wrong) in August 2019 about whether the default settings for Scikit-learn's implementation of logistic regression are misleading to new users.
+Interpretability and interoperability are required to properly evaulate the original research and to strengthen findings.
+
+## Takes time
+
+Making an analysis reproducible takes time and effort, particularly at the start of the project.
+This may include agreeing upon a [testing framework](../../testing/testing), setting up [version control](../../version_control) such as a Github repository and [continuous integration](../../continuous_integration/continuous_integration), and [managing data](../../rdm/rdm.html).
+Throughout the project, time may be required to maintain the reproducible pipeline.
+
+Time may also be spent communicating with collaborators to agree which parts of the project may be open source and when and how these outputs are shared.
+Researchers may find that they need to "upskill" their colleagues to allow the team to benefit from reproducibility tools such as git and GitHub, containers, Jupyter notebooks, or databases.
+
+```{figure} ../../figures/help-you-of-the-future.jpg
+---
+width: 500px
+name: help-you-of-the-future
+alt: A cartoon of a woman passing a folder of documents back to herself. Speech bubble says You're mainly keeping records for you in the future.
+---
+Although making clear documentation may feel like it is taking a lot of time in the moment, you are helping your and your collaborators remember what you have done so it is easy to reuse the work or make changes in the future.
+Illustration by The Ludic Group LLP from Kirstie Whitaker's keynote presentation at Scientific Data in 2017.
+Used under a CC-BY 4.0 license.
+DOI: [10.6084/m9.figshare.5577340.v1](https://doi.org/10.6084/m9.figshare.5577340.v1)
+```
+
+However, _The Turing Way_ community advocates that this time is more than made up for by the end of the project.
+Take as a thought experiment a reviewer asking for "just one more analysis" when the publication has been submitted to a journal.
+In many cases this request will come 6 to 12 months after the research team have worked with the raw data.
+It can be very hard to go back in time to find the one part of the pipline that the reviewer has asked you to change.
+If the work is fully reproducible, including with version controlled data and figure generating code, this analysis will be very fast to run and incorporate into the final research output.
+The analysis pipeline can be easily adapted as needed in response to co-author and reviewer requests.
+It can also be easily reused for future research projects.
```

Comment:
```suggestion
However, _The Turing Way_ community advocates that this time is more than made up for by the end of the project.
Take as a thought experiment a reviewer asking for "just one more analysis" when the publication has been submitted to a journal.
In many cases, this request will come 6 to 12 months after the research team have worked with the raw data.
It can be very hard to go back in time to find the one part of the pipeline that the reviewer has asked you to change.
If the work is fully reproducible, including version-controlled data and figure generating code, this analysis will be very fast to run and incorporate into the final research output.
The analysis pipeline can be easily adapted as needed in response to co-author and reviewer requests.
It can also be easily reused for future research projects.
```